sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -709,15 +802,24 @@ export class CertificatesClient {
    * ```
    * @summary Retrieves a certificate from the certificate's name (includes the certificate policy)
    * @param name The name of the certificate
-   * @param requestOptions The optional parameters
+   * @param options The optional parameters
    * @returns Promise<Certificate>
    */
   public async getCertificateWithPolicy(
     name: string,
-    requestOptions?: coreHttp.RequestOptionsBase
+    options?: RequestOptions
   ): Promise<CertificateWithPolicy> {
-    let result = await this.client.getCertificate(this.vaultBaseUrl, name, "", { requestOptions });
+    const span = this.createSpan("getCertificateWithPolicy", options);
```

Comment:
Now that this api is changed to take `RequestOptions` and not `RequestOptionsBase`, below is needed as well
```suggestion
    const span = this.createSpan("getCertificateWithPolicy", options && options.requestOptions);
```